### ISSUE TITLE
Dynamic risk matrix

### ISSUE THREAD

**Issue by pzljanb** [Puzzlepart/prosjektportalen365#433]
**Is your feature request related to a problem? Please describe.**
Not all organizations uses 5x5 risk matrix.

**Describe the solution you'd like**
A dynamic soluton where you can configure the size of the matrix.
E.g. 4x4, 6x6
This should be configurable on Hub level to maintain similarity thru the Portfolio level.

**Additional context**
In case of project by project configuration the definition used should also be identified to make it groupable on portfolio level.

**Comment by tarjeieo** [comment]
@Rundez have a look at this one, did you support this configuration as part of the changes you made in the program branch?

**Comment by Rundez** [comment]
@tarjeieo Yes, this configuration is supported. But it would need changes in the fields, and some css-tweaking to get the scaling right on most displays.

**Comment by Remi749** [comment]
Wow, this looks great @olemp !

**Comment by pzljanb** [comment]
Fantastic @olemp

**Comment by olemp** [comment]
Good to hear! Now it's much easier to do #499
